code/datums/votes/custom_vote.dm

```diff
@@ -17,23 +12,42 @@
 /datum/vote/custom_vote/reset()
 	default_choices = null
 	override_question = null
+	count_method = VOTE_COUNT_METHOD_SINGLE
 	return ..()
 
-/datum/vote/custom_vote/can_be_initiated(mob/by_who, forced = FALSE)
+/datum/vote/custom_vote/can_be_initiated(forced)
 	. = ..()
-	if(!.)
-		return FALSE
+	if(. != VOTE_AVAILABLE)
+		return .
+	if(forced)
+		return .
 
 	// Custom votes can only be created if they're forced to be made.
 	// (Either an admin makes it, or otherwise.)
-	return forced
+	return "Only admins can create custom votes."
 
 /datum/vote/custom_vote/create_vote(mob/vote_creator)
+	var/choice_method = tgui_input_list(
+		user = vote_creator,
+		message = "Single or multiple choice?",
+		title = "Choice Method",
+		items = list("Single", "Multiple"),
+		default = "Single",
+	)
+	switch(choice_method)
+		if("Single")
+			count_method = VOTE_COUNT_METHOD_SINGLE
+		if("Multiple")
+			count_method = VOTE_COUNT_METHOD_MULTI
+		else
+			to_chat(vote_creator, span_boldwarning("Unknown choice method. Contact a coder."))
```

Comment:
```suggestion
			stack_trace("Got '[choice_method]' in create_vote() for custom voting.")
			to_chat(vote_creator, span_boldwarning("Unknown choice method. Contact a coder."))
```

bit easier to debug or figure out a reproduction if the vote creator seeks out coder help